### ISSUE TITLE
Resolve button not informative and not intuitive "ours/theirs" add branch name

### ISSUE THREAD

**Issue by reslear** [sublimehq/sublime_merge#1735]
**Problem description**

Every time I do not understand which of them is "ours" and "theirs" 

<img alt="Screenshot 2023-03-13 at 13 33 26" src="https://user-images.githubusercontent.com/12596485/224703534-5abd307b-b44d-4dcc-a018-15c031a63dee.png">

**Preferred solution**

It would be more informative to add a branch name, for example.

![Screenshot 2023-03-13 at 13 37 29](https://user-images.githubusercontent.com/12596485/224708392-4bcc4c98-3612-41ef-8f06-988d6263fb37.png)

**Alternatives**

![alternative](https://user-images.githubusercontent.com/12596485/224713581-53061e27-ea07-49a9-9f18-4656f5b84eae.png)

**Screenshots**

on top

**Other editors**

Tower Intuitive write **from: branch**  
<img width="967" alt="Screenshot 2023-03-13 at 14 00 48" src="https://user-images.githubusercontent.com/12596485/224709185-a1d956bb-36be-43ce-a13a-2e2de53af038.png">

**Comment by srbs** [comment]
related: #1673, #1261
see also: #792

**Comment by cyber-distillery** [comment]
Yes, I totally agree with this, some products in the market already have such a feature or better informing about what "ours/theirs" mean. However, the reason why I have not switched is because I love the UI of Sublime Merge. Hope it will be added soon 😃

**Comment by michaelblyons** [comment]
The next version of Git adds placeholders for ours/theirs/common label names in the instructions to a mergetool.
